File: video-service/src/client/java/com/boclips/videos/service/client/ContentPartner.java
package com.boclips.videos.service.client;

import lombok.Builder;
import lombok.Data;

import java.util.Currency;

@Builder(toBuilder = true)
@Data
public class ContentPartner {
    private final ContentPartnerId contentPartnerId;
    private final String name;
    private final Boolean official;
    private final Currency currency;
}

File: video-service/src/main/kotlin/com/boclips/videos/service/domain/model/video/ContentPartner.kt
package com.boclips.videos.service.domain.model.video

data class ContentPartner(
    val contentPartnerId: ContentPartnerId,
    val name: String
) {
    override fun toString(): String {
        return "ContentPartner(id = ${this.contentPartnerId.value}, name = ${this.name})"
    }
}
